### ISSUE TITLE
[EPIC] Gatsby v2 Release

### ISSUE THREAD

**Issue by marisamorby** [gatsbyjs/gatsby#6819]
## Goal of v2

_Top three goals we're trying to accomplish with v2_

1. V2 is stable enough to release as the default package (e.g. `gatsby@latest`)
2. Migrating a site from v1 to v2 will result in `gatsby build` finishing more quickly
3. Upgrade all dependencies to the latest supported version

## Scope of v2

_What needs to be completed in order for v2 to launch?_

- [ ] Make sure all v2 issues that are not assigned to v2 project are verified and evaluated (if they should be marked as blockers)
- [ ] Remove all non-blocking issues from the scope
- [x] All blockers must be completed in order for v2 to launch
    - [To Do - v2 column are blockers](https://github.com/gatsbyjs/gatsby/projects/2)
- [x] All remaining breaking changes for v2 must be completed before we can create a release candidate (`2.0.0-rc1`)
- [x] All official plugins are updated for v2 compatibility (including docs)
- [x] A launch blog post is written and ready to publish

### Next Steps

- [x] Define Scope
- [x] Move non-blockers to the Nice to Have column
- [x] Create sub-epics
- [x] Organize issues in sub-epics
- [x] Decide which blockers must be finished before launching `rc1`
- [x] Finish blockers and launch `rc1`

## Sub-Epics
- [ ] UX & Docs
- [ ] Bug fixes
- [ ] Breaking Changes

## Resources
[v2 GitHub Project](https://github.com/gatsbyjs/gatsby/projects/2)

v2 Standup Template

[Epic RFC for v2](https://docs.google.com/document/d/191M3VQXwlkXb2qilavp4CYLligqzGU_a2djNISgr8e8/edit#heading=h.8quq53b0ewbn)

**Comment by marisamorby** [comment]
@m-allanson 

cc @pieh @KyleAMathews 

I'm not sure what the appropriate sub-epics are for these issues. Could you please move them to the appropriate sub-epic?

I have put each of these into the Gatsby v2 Release Epic for now so that they were easy to track.

### Move into sub-epics
- [ ] [Pre-release v2 ToDos](https://github.com/gatsbyjs/gatsby/issues/5953)
- [ ] [Guidelines around layout changes](https://github.com/gatsbyjs/gatsby/issues/6127)

**Comment by marisamorby** [comment]
All [ToDo -v2 Blockers](https://github.com/gatsbyjs/gatsby/projects/2) listed under the Gatsby v2 Release Project on GitHub have been moved into their corresponding Sub-Epics.

_Example: Any issue under [ToDo - v2 Blockers](https://github.com/gatsbyjs/gatsby/projects/2) that related to UX, Documentation, or Research has been listed under **[Sub-Epic] v2 Release: UX & Docs #6820**_.
